src/components/Table/TableHeaderComponent.jsx

```diff
@@ -30,8 +50,7 @@ class TableHeader extends Component {
 
   buildTableHeaders = () => {
     const {properties, visibleColumns} = this.props;
-
-    return visibleColumns.map((item, index) => {
+    let mappedVisibleColumns = visibleColumns.map((item, index) => {
```

Comment:
Those are actual Header Columns, so at least 'mappedVisibleColumns' and 'item' argument names are not justified, since item is not an item, but a column header name form schema.